build_tools/azure/test_script.sh

```diff
@@ -46,7 +46,7 @@ if [[ -n "$CHECK_WARNINGS" ]]; then
     # numpy's 1.19.0's tostring() deprecation is ignored until scipy and joblib removes its usage
     TEST_CMD="$TEST_CMD -Werror::DeprecationWarning -Werror::FutureWarning -Wignore:tostring:DeprecationWarning"
 
-    # Python 3.10 deprecates disutils and is imported by numpy interally during import time
+    # Python 3.10 deprecates distutils and is imported by numpy integrally during import time
```

Comment:
The codespell suggestion of "internally" seems more likely here.

Reply:
Ah, it makes sense after re-reading it. Will fix it.

Reply:
I've done a PR in Codespell to add your suggestion too though! https://github.com/codespell-project/codespell/pull/2316